### PR TITLE
KOGITO-7630 Added a reminder to include a card for a new guide to CON…

### PR DESCRIPTION
**Pull request by hbelmiro**
…TRIBUTING.md

**JIRA:** https://issues.redhat.com/browse/KOGITO-7630

- [x] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [x] The nav.adoc file has a link to this guide in the proper category

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -12,3 +12,4 @@ Please make sure that your PR meets the following requirements:
 - [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
 - [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
 - [ ] The nav.adoc file has a link to this guide in the proper category
+- [ ] The index.adoc file has a card for this guide in the proper category
```
Comment (ricardozanini) [inline .github/pull_request_template.md:15]: See #56

**Review by ricardozanini — APPROVED** [review]
Thank you!! But you can skip adding it to the template, we are about to merge #56.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -8,6 +8,13 @@ We accept all kinds of contributions:
 
 > To include a new guide or documentation content, please **open an issue first** so we can discuss in more detail what needs to be done. We use [JIRA](https://issues.redhat.com/) to track our tasks, so the issue is likely to turn into a JIRA issue.
 
+## Including a new guide
+
+1. Open a JIRA issue
```
Comment (hmanwani-rh) [inline CONTRIBUTING.md:13]:
```suggestion
1. Open a JIRA issue and add a sub-task for Doc review. 
```

**Review by hmanwani-rh — COMMENTED** [review]
@hbelmiro One addition regarding the doc review sub-task.

**Pull request merged**